composer.json

```diff
@@ -27,7 +27,7 @@
     "require": {
         "php": ">=7.0",
```

Comment:
Since you are using strict types bumping to 7.1 would be a good idea. It would allow for better typehinting.

Reply:
👍 Done in https://github.com/SocialConnect/auth/pull/84/commits/24f21175c27336a7b9b00a2d0d51fb3c69fce732